Library/Formula/docker-machine.rb

```diff
@@ -22,9 +22,13 @@ def install
     mkdir_p buildpath/"src/github.com/docker/"
     ln_sf buildpath, buildpath/"src/github.com/docker/machine"
 
-    system "make", "build"
-    bin.install Dir["bin/*"]
+    Dir.chdir(buildpath/"src/github.com/docker/machine/") do
+      # Hack to force ruby chdir into the symlinked path
+      ENV["PWD"] = buildpath/"src/github.com/docker/machine/"
+      system "make", "build"
+    end
```

Comment:
I think the hack is needed here due to the use of the `vendor` dependency feature.

    export GO15VENDOREXPERIMENT=1

`docker-machine` is the *only* Go package in Homebrew, that is using that, and at the same time using `make` for building.

There is one other package, namely `keybase`, using `GO15VENDOREXPERIMENT`, but it uses `go build`.